chemicaltoolbox/rdock/rbcavity.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="rbcavity" name="rDock cavity definition" version="0.1">
+    <description>- generate the active site definition needed for rDock docking</description>
+    <requirements>
+        <requirement type="package" version="2013.1-0">rdock</requirement>
+    </requirements>
+    <command><![CDATA[
+        ln -s $receptor_prm ./receptor.prm &&
+        ln -s '$receptor' receptor.mol2 &&
+        ln -s '$ligand' ligand.sdf &&
+        rbcavity -was -d -r receptor.prm &&
+        mv receptor.as '$activesite'
+    ]]></command>
+
+    <configfiles>
+        <configfile name="receptor_prm">RBT_PARAMETER_FILE_V1.00
+RECEPTOR_FILE receptor.mol2
+RECEPTOR_FLEX 3.0
+SECTION MAPPER
+    SITE_MAPPER RbtLigandSiteMapper
+    REF_MOL ligand.sdf
+    RADIUS $radius
+    SMALL_SPHERE $sphere
+    MIN_VOLUME $min_volume
+    MAX_CAVITIES 1
+    VOL_INCR $vol_incr
+    GRIDSTEP $gridstep
+END_SECTION
+SECTION CAVITY
+    SCORING_FUNCTION RbtCavityGridSF
+    WEIGHT $weight
+END_SECTION
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param type="data" name="receptor" format="mol2" label="Receptor" help="Select a receptor (Mol2 format)."/>
+        <param type="data" name="ligand" format="mol,sdf" label="Reference ligand" help="Single ligand in Mol or SDF format"/>
+        <param name="radius" type="float" value="6.0" label="Mapper sphere radius" help="Mapper radius of large sphere in angstroms; rDock will search for cavities to the small sphere but not to the large sphere"/>
+        <param name="sphere" type="float" value="1.0" label="Mapper small sphere radius" help="Mapper radius of small sphere in angstroms; rDock will search for cavities accessible to the small sphere but not to the large sphere"/>
+        <param name="min_volume" type="integer" value="100" label="Mapper minimum volume" help="Mapper minimum volume in cubic angstroms. rDock will ignore cavities which are smaller than this value"/>
+        <param name="vol_incr" type="float" value="0.0" label="Mapper volume increment" help="Mapper volume increment in angstroms. When excluding the volume occupied by the receptor from the cavity search, rDock will temporarily increase the radius of receptor atoms by this amount"/>
+        <param name="gridstep" type="float" value="0.5" label="Mapper grid step" help="Grid resolution for mapping in angsroms"/>
+        <param name="weight" type="float" value="1.0" label="Cavity weight" help="Cavity weight"/>
+    </inputs>
+    <outputs>
+        <data name="activesite" format="binary" label="rbcavity active site on ${on_string}"/>
```

Comment:
let's create a proper filetype for this: https://github.com/galaxyproject/galaxy/pull/8727

Reply:
OK. looks good.